include/avif/internal.h

```diff
@@ -59,6 +59,17 @@ static inline void avifBreakOnError()
         }                                 \
     } while (0)
 
+// Same as AVIF_CHECKERR() but also assert() in debug build configuration.
+// Can be used instead of assert() for extra security in release builds.
+#define AVIF_ASSERT_OR_RETURN(A, ERR) \
```

Comment:
Delete the `ERR` parameter. This macro should always use `AVIF_RESULT_INTERNAL_ERROR`.

Reply:
> This macro should always use `AVIF_RESULT_INTERNAL_ERROR`.

I agree but at calling site it is not as readable in my opinion. What about a `static_assert()` to make sure `ERR` is always `AVIF_RESULT_INTERNAL_ERROR`?

Reply:
I see what you mean. Here is an example:

```
    AVIF_ASSERT_OR_RETURN(layerCount == 3)
```

I think this is caused by the macro name. Perhaps we can rename the macro `AVIF_ASSERT_OR_RETURN_ERROR` or even `AVIF_ASSERT_OR_RETURN_INTERNAL_ERROR`.

Adding the `ERR` parameter back is also fine by me.

`static_assert` is only available in C11 or later, so we will need to add an `AVIF_STATIC_ASSERT` macro if we want to use `static_assert` in our C code.